third_party/VENDOR-LICENSE/github.com/hashicorp/golang-lru/2q.go

```diff
@@ -0,0 +1,223 @@
+package lru
```

Comment:
<!--copyright-bot-->
Missing copyright header:
```suggestion
/*
Copyright 2020 The Knative Authors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

package lru
```

Reply:
oh boy 🤔 

I'll whack this bot too

Reply:
which one of you is the bot :D